index.css

```diff
@@ -30,16 +30,34 @@ h1{
     font-weight: bold;
 }
 button {
-    background: #3d9cb1;
-    color: #0b2b44;
-    padding: 10px 20px;
-    border: 1px solid #3d9cb1;
-    font-weight: bold;
+    box-sizing: border-box;
+    appearance: none;
+    background-color: transparent;
+    border: 2px solid #3D9CB1;
+    border-radius: .6em;
+    color: #3D9CB1;
+    cursor: pointer;
+    font-size: .7rem;
+    line-height: 1;
+    margin: 5px;
+    padding: 1em 1.7em;
+    text-decoration: none;
+    text-align: center;
+    text-transform: uppercase;
 }
 
-#delete-btn {
-    background:#3d9cb1;
-    color: #0b2b44;
+button:hover,
+button:focus {
+    color: #FFF;
+    outline: 0;
+}
+
+.primary {
+    transition: box-shadow 300ms ease-in-out, color 300ms ease-in-out;
```

Comment:
What do you think about speeding up the transition on all of the button effects. I don't mind the mouseover effect, it just seems slow to me.

Reply:
I speed up the buttons too. I think 100ms is fast enough, anything less than that doesn't feel like animation. Unless you don't want animation at all :))